folding-schemes/src/folding/nova/decider_eth_circuit.rs

```diff
@@ -419,13 +419,13 @@ where
         )?;
         let zero_y = NonNativeFieldVar::<C1::BaseField, C1::ScalarField>::new_constant(
             cs.clone(),
-            C1::BaseField::one(),
+            C1::BaseField::zero(),
         )?;
         u_i.cmE.x.enforce_equal(&zero_x)?;
         u_i.cmE.y.enforce_equal(&zero_y)?;
```

Comment:
same as in [the other `zero_y` comment](https://github.com/privacy-scaling-explorations/sonobe/pull/84/files#r1559713822), maybe now can just use a single var for `zero` for `cmE.x` & `cmE.y`?